### ISSUE TITLE
Invalid vmArgument merging when launch.json has a list of vmArgs

### ISSUE THREAD

**Issue by kdvolder** [microsoft/vscode-spring-boot-dashboard#95]
See: https://github.com/spring-projects/sts4/issues/395#issuecomment-568018685

Summary:

If we make a launch.json that has something like:

```
           "vmArgs": [ "-Dspring.jmx.enabled=true", "-Dspring.application.admin.enabled=true" ]
 ```

Then this launches fine using the vscode java debugger tooling directly. 

However when we launch it from the Boot Dashboard then the arguments are joined together with a `,` in between. This changes the meaning of the arguments and causes apps launched this way to malfunction.

The culprit I think is here:

https://github.com/microsoft/vscode-spring-boot-dashboard/blob/master/src/Controller.ts#L74

I guess javascript implicitly converts the array into a string adding commas between.

**Comment by Eskibear** [comment]
Yes, we can directly pass an array of vmArgs to the debugger extension, which has been supported since a couple of months ago. 
Thank you for the fix!

**Comment by Eskibear** [comment]
shipped with v0.1.8
